apis/Google.Cloud.Spanner.Admin.Database.V1/Google.Cloud.Spanner.Admin.Database.V1/DatabaseAdminClient.cs

```diff
@@ -1930,12 +1930,12 @@ public DatabaseAdminClientImpl(DatabaseAdmin.DatabaseAdminClient grpcClient, Dat
         /// <returns>
         /// A Task containing the RPC response.
         /// </returns>
-        public override async Task<Operation<Empty>> UpdateDatabaseDdlAsync(
+        public override async Task<Operation<Empty, UpdateDatabaseDdlMetadata>> UpdateDatabaseDdlAsync(
             UpdateDatabaseDdlRequest request,
             CallSettings callSettings = null)
         {
             Modify_UpdateDatabaseDdlRequest(ref request, ref callSettings);
-            return new Operation<Empty>(
+            return new Operation<Empty, UpdateDatabaseDdlMetadata>(
                 await _callUpdateDatabaseDdl.Async(request, callSettings), LongRunningOperationsClient);
```

Comment:
Well spotted - this is in the code generator, so I've filed https://github.com/googleapis/toolkit/issues/1199